### DISCUSSION TITLE
status of Dominican Rite implementation?

### DISCUSSION THREAD

**Discussion by Geremia** [DivinumOfficium/divinum-officium#2564] [General]
What is the source for the Dominican Rite rubrics? Is it still a WIP?
[web/www/horas/Help/rubrics.html](https://github.com/DivinumOfficium/divinum-officium/blob/master/web/www/horas/Help/rubrics.html) doesn't discuss the Dominican Rite.

Currently, "Ordo Praedicatorum" doesn't display a Dominican Rite Mass.

What else needs to be implemented so everything (office hours & Mass) is of the Dominican Rite when "Ordo Praedicatorum" is selected?

**Comment by malukenho** [comment]
I think we only have the "Complines" in place, we still have a lot of work ahead.

**Comment by Geremia** [comment]
[Breviarium S.O.P.](https://breviariumsop.blogspot.com/p/the-1909-breviarium-juxta-ritum-sacri.html) blog:

> For accuracy of text, clarity of rubrics, and convenience of arrangement, \[[**1909 _Breviarium S.O.P._ by Cormier, O.P.**](https://archive.org/details/BrevariumIuxtaRitumSacriOrdinisPraedicatorum)\] was the finest edition of the Dominican breviary ever published. The Order was indeed fortunate to receive from this painstaking scholar the fruits of nearly forty years of research.

—[_The History of the Dominican Liturgy_](https://isidore.co/calibre/#book_id=6147&panel=book_details), Fr. William Bonniwell, O.P., 1944, p. 362

We could use Cormier's edition to implement the rest of the Dominican Rite office.

**Comment by Geremia** [comment]
@malukenho I see you've created
```
web/www/horas/Latin/Ordinarium/CompletoriumOP.txt
web/www/horas/Latin/Ordinarium/LaudesOP.txt
web/www/horas/Latin/Ordinarium/MatutinumOP.txt
web/www/horas/Latin/Ordinarium/NonaOP.txt
web/www/horas/Latin/Ordinarium/PrimaOP.txt
web/www/horas/Latin/Ordinarium/SextaOP.txt
web/www/horas/Latin/Ordinarium/TertiaOP.txt
web/www/horas/Latin/Ordinarium/VesperaOP.txt
```
now, though they seem to be a placeholder (not those of [Cormier's 1909 edition mentioned above](#discussioncomment-1073821)).

It would be good to implement a Dominican Parvum B.M.V., too, maybe based off the [_Officium B. Mariæ Virginis juxta ritum Sac. Ord. Fr. Pædicatorum_](https://bc.dominikanie.pl/dlibra/docmetadata?id=215) (1894) ed. by Frühwirth, O.P.

**Reply by malukenho** [reply]
To be honest I don't know. I personally only pray the complines following the FSVF (they have a live stream); So I just follow what they follow.

I can help with code, but not with the logic here since I'm uneducated on breviarium structure.

**Reply by DowryOfMary** [reply]
I've just purchased an original Officium B. Mariæ Virginis juxta ritum Sac. Ord. Fr. Pædicatorum, and so I will work on adding it to DO.

Many Thanks.

**Comment by Geremia** [comment]
@malukenho The "logic" is documented in [`web/www/horas/Help/technical.html`](https://htmlpreview.github.io/?https://github.com/DivinumOfficium/divinum-officium/blob/master/web/www/horas/Help/technical.html).

**Comment by ThomasMariaOP** [comment]
Why Cormier's edition will be used to implement rubrics and calendar? Of course, it's the best version of the Dominican Rite, but it's not actual, so it's unuseful when someone wants to pray the breviary in union with the currently celebrated liturgy. I think Divinum Afflatu or 1962 versions will be more appropriate at the beginning, and then other versions could be implemented. I pray every day 1962 version of Dominican Rite breviary and I have both parts at home, but of course, I'll help with any version that will be implemented.

**Reply by Geremia** [reply]
> Why Cormier's edition will be used to implement rubrics and calendar?

I'm only suggesting Cormier's.
I think the current Dominican compline that @malukenho et al. have implemented is according to the 1962.
